utils/vlog/tests/muxes/proper.golden.pb_type.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version='1.0' encoding='utf-8'?>
+<pb_type xmlns:xi="http://www.w3.org/2001/XInclude" name="use_mux" num_pb="1">
+  <input name="a" num_pins="1"/>
+  <input name="b" num_pins="1"/>
+  <input name="cin" num_pins="1"/>
+  <output name="cout" num_pins="1"/>
+  <pb_type name="AMUX" num_pb="1">
```

Comment:
```suggestion
  <pb_type name="amux" num_pb="1">
```

Reply:
I think this file should be removed. It's name suggests that this is how the golden pb_type should look like, but it is nowhere near to what is modelled in verilog.

Reply:
?